app/code/core/Mage/Catalog/Model/Product/Image.php

```diff
@@ -73,16 +72,14 @@ class Mage_Catalog_Model_Product_Image extends Mage_Core_Model_Abstract
      */
     protected $_newFile;
 
-    /**
-     * @var Varien_Image
-     */
-    protected $_processor;
+    /** @var \Intervention\Image\Interfaces\ImageInterface */
+    protected $image;
 
     /**
      * @var string e.g. "small_image"
      */
     protected $_destinationSubdir;
-    protected $_angle;
+    protected float $_angle;
```

Comment:
```suggestion
    protected float $_angle = 0;
```

Fatal error:  Uncaught Error: Typed property Mage_Catalog_Model_Product_Image::$_angle must not be accessed before initialization in /shared/httpd/github-maho/app/code/core/Mage/Catalog/Model/Product/Image.php:324